engine/language-server/src/main/scala/org/enso/languageserver/search/SuggestionsHandler.scala

```diff
@@ -439,14 +410,14 @@ final class SuggestionsHandler(
 
     case SuggestionUpdatesCompleted =>
       if (state.suggestionUpdatesQueue.nonEmpty) {
-        self ! SuggestionUpdatesBatch(state.suggestionUpdatesQueue.removeAll())
+        self ! state.suggestionUpdatesQueue.dequeue()
```

Comment:
Are we giving up on "batch updates" dequeuing one by one again?

Reply:
Libraries are still loaded in batch. There is no benefit in batching other suggestions. There's too little of them